.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
-before_install: npm i -g npm@1.4.28
 node_js:
-  - 0.8
\ No newline at end of file
+  - 0.10
```

Comment:
I'm adding this to compensate for the lack of Node 0.8 support. I assume the jump from v0.8 to v0.10 would be much better than upgrading to v4.